lib/chef/knife/cloud/ovirt_service.rb

```diff
@@ -18,6 +18,7 @@ def initialize(options = {})
                                 ovirt_username: Chef::Config[:knife][:ovirt_username],
                                 ovirt_password: Chef::Config[:knife][:ovirt_password],
                                 ovirt_url: Chef::Config[:knife][:ovirt_url],
+                                ovirt_ca_no_verify: true
```

Comment:
This should be a configuration option, if it's not already. Some folks run there web services with valid SSL configuration and want the verification.

Reply:
The other thought is to make another pull request allowing users to override CA verification.